site/docs/tutorials/cdc-join.md

```diff
@@ -0,0 +1,718 @@
+---
+title: Join Change Data Capture Records
+sidebar_label: Join
+description: How to maintain a joined image of Change Data Capture data from multiple tables.
+---
+
+**Change data capture** refers to the process of **observing changes
+made to a database** and extracting them in a form usable by other
+systems, for the purposes of replication, analysis and many more.
+
+Change Data Capture is especially important to Jet, because it allows
+for the **streaming of changes from databases**, which can be
+efficiently processed by Jet.
+
+Implementation of CDC in Jet is based on
+[Debezium](https://debezium.io/), which is an open source distributed
+platform for change data capture.
+
+In our previous tutorials on how to extract change event data from
+[MySQL](cdc-mysql.md) and [Postgres](cdc-postgres.md) databases we've
+seen how to monitor single database tables and how maintain an
+up-to-date image of them in memory, as a map.
+
+Here we will examine how to make our map hold enriched data, combined
+(joined, if you will) from multiple tables. Let's say customers and all
+the orders belonging to them, as one entity.
+
+## 1. Install Docker
+
+This tutorial uses [Docker](https://www.docker.com/) to simplify the
+setup of a PostgreSQL database, which you can freely experiment on.
+
+1. Follow Docker's [Get Started](https://www.docker.com/get-started)
+   instructions and install it on your system.
+2. Test that it works:
+   * Run `docker version` to check that you have the latest release
+     installed.
+   * Run `docker run hello-world` to verify that Docker is pulling
+     images and running as expected.
+
+## 2. Start Database
+
+In this tutorial we will provide all code and configs for MySQL, but
```

Comment:
Seems to contain both?

Reply:
Yeah, I was thorn between covering both or focusing on one of them and this is a left-over. Will fix.